scripts/upgrade

```diff
@@ -23,6 +23,7 @@ export on_calendar="$(ynh_app_setting_get $app on_calendar)"
 export conf="$(ynh_app_setting_get $app conf)"
 export data="$(ynh_app_setting_get $app data)"
 export apps="$(ynh_app_setting_get $app apps)"
+export mailalert="$(ynh_app_setting_get $app mailalert)"
```

Comment:
We probably need to configure errors_only by default ?

Reply:
Yes to me the best would be to have `errors_only` the default. Whether the question is asked in the manifest or it's just a hidden setting, both options seem good to me.
I don't know what would be the best with bash to let mailalert default to some value in the upgrade script. If you want to do it go ahead, otherwise I can just do a simple `if ... then ... fi`.

Reply:
The commit below adds a default value for mailalert setting in the upgrade script.
In my previous commit, the backup script was already defaulting to sending mail only on errors, if the setting was missing, but it's cleaner like this I guess.
I did it just with a simple `if then fi`, don't know if there's a cleaner way to do it, but it should be ok like this I guess.

Reply:
I marked this conversation as resolved, first time I use this github feature :/ , so feel free to reopen it if makes sense to you.